source/index.html.md

```diff
@@ -426,10 +428,51 @@ Example response (snippet):
         "code": 20000
     },
     "processed_at": "2018-07-09T12:58:56+00:00",
-    "threed_secure": true
+    "3dsecure": {
+        "version": "1.0.2",
+        "status": "Y"
+    }
+}
+````
+
+To perform a 3-D Secure version 2 transaction you make an ordinary
+authorization including an `ares` or an `rreq` value. The former is used in the
+following example:
+
+````shell
+curl -X POST https://gateway.test.clearhaus.com/authorizations \
+     -u <your-api-key>: \
+     -d "amount=2050"   \
+     -d "currency=EUR"  \
+     -d "ip=1.1.1.1"    \
+     -d "card[pan]=4111111111111111" \
+     -d "card[expire_month]=06"      \
+     -d "card[expire_year]=2022"     \
+     -d "card[csc]=123"              \
+     --data-urlencode "card[3dsecure][v2][ares]=<some-ares-value>" \
+     -H "Signature: <signing-api-key> RS256-hex <signature>"
+````
+
+Example response (snippet):
+
+````json
+{
+    "id": "d0949241-1ee8-47da-a77c-d251fd9e1e88",
+    "status": {
+        "code": 20000
+    },
+    "processed_at": "2020-07-03T11:06:58+00:00",
+    "3dsecure": {
+        "version": "2.1.0",
+        "status": "Y"
```

Comment:
Doesn't this property require an explanation?

Reply:
Thanks for asking!

In 3DSv1 it's the `TX.status` and in 3DSv2 it's the `transStatus` — IMO it's pretty unambiguous in the context of 3-D Secure. Interpreting the value is not exactly trivial though; even from 3DSv2.1.0 to 3DSv2.2.0 new values are being introduced.

It is our plan to add "higher level" response values like "`sca`" and "`liability_shift`" 🙂

Together with these, one can notice downgrades from the `status`. However, perhaps we should just remove this `status` altogether and respond a value like "`downgraded`", also because the value was already sent in the request.

Reply:
On one hand I agree that `status` is superfluous, because it was sent in the request, however, the same argument can be made for `version` (at least `2.x.y`), and then the dictionary becomes empty :man_shrugging: :thinking:

Reply:
Perhaps we should start out with just `3dsecure: { downgraded: true/false }` (only included for 3-D Secured transactions).

Reply:
And of course explain `downgraded`. AFAIK, it is always a downgrade to no 3DS at all, e.g. not from `Y` to `A`.

Reply:
Downgrades can also happen for tokenized transactions 🙈

Reply:
We have decided elsewhere that the `3dsecure` dict will remain as-is for now.